resources/Dockerfile

```diff
@@ -155,3 +157,9 @@ RUN if [ "$TARGETARCH" = "amd64" ]; then \
     && unzip ethr_linux.zip \
     && cp ethr /usr/local/bin \
     && rm ethr_linux.zip; fi
+
+# ENTRYPOINT script that only (as of this commit) starts a D-Bus session
+# and runs the commands passed to it
+COPY docker_entrypoint.sh /etc/init.d
+RUN chmod +x /etc/init.d/docker_entrypoint.sh
+ENTRYPOINT ["/etc/init.d/docker_entrypoint.sh"]
```

Comment:
I don't think it is a good idea, as many container instances do not need to use D-Bus sessions. If the purpose was for making integration tests easier, you can always setup the D-Bus session from the integration test script.

Reply:
Good point, I'm not sure what my rationale was at the time.